src/main/java/echoquery/utils/SlotUtil.java

```diff
@@ -178,6 +189,61 @@
         "below or equal",
         "is below or equal"
     }));
+    refineAddExpr.addAll(Arrays.asList(new String[]{
+        "and",
+        "and if",
+        "and when",
+        "and where",
+        "and also",
+        "and also where",
+        "also",
+        "also if",
+        "also when",
+        "also where",
+        "add",
+        "add where",
+        "also add",
+        "also add where"
+    }));
+    refineReplaceExpr.addAll(Arrays.asList(new String[]{
+        "what if",
+        "what if we",
+        "what if where",
+        "how about",
+        "how about where",
+        "how about if",
+        "instead",
+        "instead how about",
+        "instead where",
+        "instead how about where",
+        "actually",
+        "actually how about where"
+    }));
+    refineDropExpr.addAll(Arrays.asList(new String[]{
+        "drop",
+        "drop where",
+        "drop the",
+        "drop the part",
+        "drop the part where",
+        "drop the clause",
+        "drop the clause where",
+        "forget",
+        "forget the",
+        "forget about",
+        "forget where",
+        "forget about where",
+        "forget the part",
+        "forget the part where",
+        "forget the clause",
+        "forget the clause where",
+        "remove",
+        "remove the",
+        "remove where",
+        "remove the part",
+        "remove the part where",
+        "remove the clause",
+        "remove the clause where"
+    }));
```

Comment:
Yeah or made a special json version of custom slots that we can read from Java, and compile into what Amazon needs. pipe dreams.